pipeline-module/README.md

```diff
@@ -90,32 +99,35 @@ Check on the GitHub site for the repository to make sure that the initial “Act
 ### 7. Add the initial content
 
 If all is well, continue working on on your local repository clone to add the actual module text and any figure files that are needed.
-Start by creating a branch for this work:
+
+a. Start by creating a branch for this work:
 
 ```shell
 git checkout -b initial-text
 ```
 
-Then copy over the content from the original document.
+b. Copy over the content from the original document.
 Replace any content **between** `\maketitle` and `\bibliography` with the original document's content.
 **Leave the \bibliography{spherex} from the template rather than using the one from the original document.**
 
 [Learn more about the spherex document class and spherex bibliography file.](https://github.com/SPHEREx/spherex-tex)
 
-Copy any images from the original document on Overleaf into the new repo.
+c. Set the correct current document version number in the `\version{}` macro.  For the initial load into GitHub, *we are keeping the version numbers as they last appeared on Overleaf.*
+
+d. Copy any images from the original document on Overleaf into the new repo.
 
-If the original document used `\modulename` in the body of the document to substitute in the name of the module, you will need to use `\themodulename` for this instead.
-(`\modulename{name}` is now the setter for the name, rather than the getter.)
+e. If the original document used `\modulename` in the body of the document to substitute in the name of the module, you will need to use `\themodulename` for this instead.
+(`\modulename{name}` is now the setter for the name, rather than the getter, which is now `\themodulename`.)
 
 If you'd like, you can compile the document locally.
 See the [spherex-tex](https://github.com/SPHEREx/spherex-tex) repository's [README](https://github.com/SPHEREx/spherex-tex/blob/main/README.md) for instructions.
 
-Once ready, push this content to GitHub:
+f. Once ready, push this content to GitHub:
 
 ```shell
 git add .
 git commit
-git push -u
+git push -u origin initial-text
```

Comment:
@jonathansick I don't know if bare `git push -u` does the right thing if one has certain config options set, but for us it doesn't seem to work.

Reply:
Ah, yes, the configuration to make that work is:

```
[push]
	default = current
```

Going back to your recommended change, I think it should be:

```
git push -u origin main
```

That way the `main` branch of the local repository corresponds to the `main` branch of the GitHub origin.